manifests/s/SublimeHQPtyLtd/SublimeText/4/Dev/4.0.0.413800/SublimeHQPtyLtd.SublimeText.4.Dev.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: SublimeHQPtyLtd.SublimeText.4.Dev

```

```suggestion
PackageIdentifier: SublimeHQ.SublimeText.4.Dev
```
SublimeHQ already exists as a publisher and should be used